### PR TITLE
chore: "Advanced page" design improvements

### PR DESCRIPTION
**Pull request by elliott-king**
### To test:
Visit "advanced page". Verify you can see table. Should be scrollable and filterable by bills & legislators. Should show legislator bill percentage on left.

Some MVP needed improvements to make the page usable:

- [x] column header - use printno & show title on hover
- [x] horizontal axis text box filter bill
- [x] vert axis text box filter legislator
- [x] css improvements: clearer divisions
- [x] show column header even while scrolled
- [x] show leg name even while scrolled
- [x] campaign super-header
- [x] all leg pct & climate leg pct

### CODE REVIEW

**Comment by agreffard** [comment]
@elliott-king When I open the advanced page I have a lot of errors like this and the app crashes:

`TypeError: billCampaignMappings[bill.basePrintNo] is not iterable`

And when I fix the frontend to handle these cases with default values it seems like I have incomplete data:

image

I dont see any network/requests error though.
Am I missing something (some setup steps?) to have  it running locally?

**Comment by elliott-king** [comment]
@agreffard looking into this, may ask to chat with you next week

**Comment by elliott-king** [comment]
@agreffard can we chat before/after the next meeting?

**Comment by agreffard** [comment]
@elliott-king Sure! I can chat tonight (Monday) or after Wednesday's meeting

**Review by agreffard — APPROVED** [review]

File: frontend/src/components/AdvancedPage/AdvancedPage.jsx [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+              campaigns={campaigns}
+            />
+          </tr>
+        </thead>
+        <tbody>
+          {filteredSenators.map((senator) => (
+            <LegislatorRow
+              key={senator.memberId}
+              member={senator}
+              bills={billPairs.map((pair) => pair[0])}
```
[the remark this answers is not in the document]
Comment (elliott-king) [reply frontend/src/components/AdvancedPage/AdvancedPage.jsx:208]: I'm not a fan of this, feels deceptive for future readers.

File: frontend/src/components/AdvancedPage/utils.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    });
+  return billSponsors;
+};
+
+/** Combine bills with the corresponding one in the other chamber */
+export const collectBillPairs = (senateBills, assemblyBills) => {
+  const titles = Object.values(senateBills).map((bill) => bill.title);
+  const senateByTitles = Object.fromEntries(
+    Object.values(senateBills).map((bill) => [bill.title, bill])
+  );
```
[the remark this answers is not in the document]
Comment (elliott-king) [reply frontend/src/components/AdvancedPage/utils.js:77]: I'm also not a fan of `reduce` but I'll make this change.

**Comment by elliott-king** [comment]
@agreffard made the changes, merging.

**Pull request merged**